cmd/diag/sysinfo/sysinfo.go

```diff
@@ -61,15 +63,64 @@ func collectInfo(cliCtx *cli.Context) error {
 		util.RenderError(err)
 	}
 
+	stringToSave := "Disk info:\n"
```

Comment:
can use strings.Builder + fmt.Fprintf for such patterns when you make many concats. For very big strings it reduces allocation footprint vastly.

Reply:
done